shell/src/main/java/alluxio/cli/fs/FileSystemShell.java

```diff
@@ -83,7 +83,7 @@ protected String getShellName() {
   }
 
   @Override
-  protected Map<String, Command> loadCommands() {
+  public Map<String, Command> loadCommands() {
```

Comment:
if `public` is required now, can you add a comment reasoning why? And if it's only for testing purposes, add `@VisibleForTesting`